### PR TITLE
Refactoring InboxView controller

### PR DESCRIPTION
**Pull request by Kharchevskyi**
https://github.com/FlowCrypt/flowcrypt-ios/issues/58

- Using Promises instead of RXSwift
- Added batch fetching
- Implemented Texture UI for Inbox

### CODE REVIEW

File: FlowCrypt/Common/Taptic.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//  Taptic.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 24.09.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+
+enum TapTicFeedback {
```
Comment (tomholub) [inline FlowCrypt/Common/Taptic.swift]: What does TapTic mean and what is it used for? Thanks!

Comment (Kharchevskyi) [reply FlowCrypt/Common/Taptic.swift]: for iPhones started I think from 8 - they edit taptic feedback when you do some actions. For example on. iPhone X when you tap on button you will have small feedback.

**Comment by Kharchevskyi** [comment]
Do you have step to reproduce for black screen?

**Comment by tomholub** [comment]
Just delete the app, reinstall it, then set it up by recovering from
backup. It seemed like the transition from backup screen to inbox was
broken.

**Pull request closed without merging**